pymc/distributions/multivariate.py

```diff
@@ -2845,3 +2844,12 @@ def zerosumnormal_logp(op, values, normal_dist, sigma, support_shape, **kwargs):
     )
 
     return check_parameters(out, *zerosums, msg="mean(value, axis=n_zerosum_axes) = 0")
+
+
+@_transformed_logprob.register(ZeroSumNormalRV, ZeroSumTransform)
+def transformed_zerosumnormal_logp(op, transform, unconstrained_value, rv_inputs):
+    _, sigma, _ = rv_inputs
+    zerosum_axes = transform.zerosum_axes
+    if len(zerosum_axes) != op.ndim_supp:
```

Comment:
I don't think this check is necessary?

Reply:
It was a sanity check. A user could have defined an invalid transform manually. I don't really care either way